docs/topics/testing/tools.txt

```diff
@@ -125,6 +125,11 @@ Use the ``django.test.Client`` class to make requests.
     The ``enforce_csrf_checks`` argument can be used to test CSRF
     protection (see above).
 
+    The ``json_encoder`` argument allows you to set a custom JSON encoder.
```

Comment:
A `.. versionchanged::` annotation and release notes for docs/releases/2.1.txt are missing.

Reply:
Done

Reply:
Done